_redirects

```diff
@@ -505,6 +505,9 @@
 /v1.6/* https://docs.tigera.io/archive/v1.6/:splat 301
 /v1.5/* https://docs.tigera.io/archive/v1.5/:splat 301
 
+# Redirect to the root folder of Calico Eneterprise
+/calico-enterprise/ https://docs.tigera.io/calico-enterprise/ 301
```

Comment:
```suggestion
/calico-enterprise/ https://docs.tigera.io/calico-enterprise/latest/about-calico-enterprise 301
```
Always helpful to be specific and save an extra redirect.

Reply:
Done